scheduler/test/cook/test/kubernetes/compute_cluster.clj

```diff
@@ -68,7 +68,8 @@
         (let [compute-cluster (kcc/->KubernetesComputeCluster nil "kubecompute" nil nil
                                                               (atom {}) (atom {}) (atom {}) (atom {}) (atom {}) (atom {}) (atom {}) (atom nil)
                                                               {:kind :static :namespace "cook"} nil nil nil nil
-                                                              (Executors/newSingleThreadExecutor))
+                                                              (Executors/newSingleThreadExecutor)
```

Comment:
Discussed using chime instead of making another executor.

Reply:
updated